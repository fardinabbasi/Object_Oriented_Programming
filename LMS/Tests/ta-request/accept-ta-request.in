POST login ? id 0 password UT_account
POST course_offer ? course_id 1 professor_id 810201475 capacity 60 time Saturday:14-16 exam_date 1403/4/3 class_number 5
POST course_offer ? course_id 7 professor_id 810201475 capacity 80 time Monday:13-15 exam_date 1403/4/6 class_number 7
POST logout ?
POST login ? id 810201475 password ry
POST post ? title "start" message "hey"
POST ta_form ? course_id 1 message "need TA for AP"
POST ta_form ? course_id 2 message "need TA for compiler"
POST logout ?
POST login ? id 810101551 password hehe
POST ta_request ? professor_id 810201475 form_id 2
POST logout ?
POST login ? id 810201475 password ry
POST close_ta_form ? id 2
accept
POST close_ta_form ? id 1
POST logout ?
POST login ? id 810101551 password hehe
GET notification ?
POST logout ?