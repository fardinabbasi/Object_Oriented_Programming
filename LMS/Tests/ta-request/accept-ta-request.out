OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
We have received 1 requests for the teaching assistant position
810101551 parisa 4: 
Not Found
OK
OK
1 Advanced Programming: Your request to be a teaching assistant has been accepted.
810201475 khousravi: New Course Offering
810201475 khousravi: New Course Offering
OK
